packages/bridge/src/reactBridge.ts

```diff
@@ -18,12 +18,46 @@ export let GarfishContext = null;
 //   // ignore
 // }
 
+type RequiredOpts = {
+  React: any;
+  ReactDOM: any;
+};
+
+type OptionalOpts = {
+  renderType: Function | string;
```

Comment:
renderType should be `"createRoot" | "unstable_createRoot" | "createBlockingRoot" | "unstable_createBlockingRoot" | (()=> "createRoot" | "unstable_createRoot" | "createBlockingRoot" | "unstable_createBlockingRoot")`